payment_gateway/views/gateway_transaction_view.xml

```diff
@@ -1,139 +1,139 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<openerp>
+<odoo>
     <data>
```

Comment:
I think we don't need `<data>` here. No more used in v10 (deprecated for the future). Please fix indent of the xml